api/controllers/places.js

```diff
@@ -6,16 +6,20 @@ const _ = require('underscore'),
       PLACE_EDITABLE_FIELDS = ['name', 'parent', 'contact', 'place_id'],
       PLACE_TYPES = ['national_office', 'district_hospital', 'health_center', 'clinic'];
 
-const minify = contact => {
-  if (!contact) {
-    return contact;
+// Minifies things you would attach to another doc:
+//   doc.parent = minify(doc.parent)
+// Not:
+//   minify(doc)
+const minify = place => {
+  if (!place || !place._id) {
+    return place;
   }
-  const result = { _id: contact._id };
+  const result = { _id: place._id };
   let minified = result;
-  while(contact.parent) {
-    minified.parent = { _id: contact.parent._id };
+  while(place.parent && place.parent._id) {
```

Comment:
Potentially not needed anymore, but more aggressively only recognises parents that are "real"